interface/modules/zend_modules/module/Carecoordination/view/carecoordination/carecoordination/upload.phtml

```diff
@@ -262,6 +262,13 @@ echo $this->headScript()->prependFile($this->basePath() . '/js/autosuggest/autos
     async function handleFileUpload(files, obj) {
         let fnames = '';
         let uploadURL = "<?php echo $this->basePath();?>/carecoordination/upload";
+        // flash to let user know we're working.
+        let f = document.getElementById('uploadFlash');
+        f.innerText = <?php echo js_escape(xlt("Starting Upload Process.")); ?>;
```

Comment:
would do (xlj for escaping in javascript context and then the jsText to escape since it is going to be printed into the html context):
```
f.innerText = jsText(<?php echo xlj("Starting Upload Process."); ?>);
```

Reply:
can't use jsText or would have. not in scope.

Reply:
Since its in javascript land can't you just use 
```
top.xlj("Starting Upload Process.");
```
Since the translation functions are in javascript?

Reply:
I think your right about the header class issue. I remember having that problem before, attempted to fix it and walked angrily away after reverting my code.